integrations/issue_test.go

```diff
@@ -139,7 +139,7 @@ func testNewIssue(t *testing.T, session *TestSession, user, repo, title, content
 	htmlDoc = NewHTMLParser(t, resp.Body)
 	val := htmlDoc.doc.Find("#issue-title").Text()
 	assert.Equal(t, title, val)
-	val = htmlDoc.doc.Find(".comment-list .comments .comment .render-content p").First().Text()
```

Comment:
thx for updates, I removed this class couse it's setting `max-width: 650px;` (semantic) later it was overridden with gittea styles to 100% but it's not correct, but it seems that `max-width: inherit;` is working as expected so I'll recheck in few more browsers and reintroduce that `comments` class

Reply:
https://github.com/fomantic/Fomantic-UI/pull/1400#event-3205584211

Reply:
Thats ok imo we aren't really using the comments class here as it is intended by fomantic since this is more of a timeline of various events than a comment response thread on a blog

Reply:
fomantic can be customized by https://fomantic-ui.com/usage/theming.html so we should be able to override this variable like https://github.com/Sorien/gitea/commit/7afe93778cace1b815bc8ff477ca145c3758479c to my knowledge it should be working but it's not :/

Reply:
Fomantic CSS is currently not being custom-built but copied because of a outstanding issue, see https://github.com/go-gitea/gitea/pull/10655. We'll be able to custom-build again with their next release, but still waiting on it.

Reply:
Or to be slightly more correct: Custom build does run but we copy the original CSS file over the custom version with that workaround. JS is still kept as custom build.